Assets/Scripts/Localization/LocalizationManager.cs

```diff
@@ -0,0 +1,99 @@
+using System.Collections.Generic;
+using PotentialRobot.Localization.Components;
+using UnityEngine;
+
+namespace PotentialRobot.Localization
+{
+    public class LocalizationManager
+    {
+        public const string c_currentLanguageKey = "CurrentLanguage";
+        private static LocalizationManager _instance;
+        public static LocalizationManager Instance
+        {
+            get
+            {
+                if (_instance == null)
+                {
+                    _instance = new LocalizationManager();
+                    _instance.Initialize();
+                }
+                return _instance;
+            }
+        }
+
+        public delegate void LanguageChanged(string language);
+        private LanguageChanged _onLanguageChanged;
+        private Dictionary<string, string> _localizations;
+        private string _previousLanguage;
+        public string CurrentLanguage { get; private set; }
+
+        public void Initialize()
+        {
+            if (PlayerPrefs.HasKey("CurrentLanguage"))
+                CurrentLanguage = PlayerPrefs.GetString("CurrentLanguage");
```

Comment:
Looks like you meant to use the `c_currentLanguageKey `const here?

Reply:
Changed to use the c_currentLanguageKey constant string